.gitignore

```diff
@@ -32,9 +32,5 @@ jspm_packages/
 .eslintcache
 
 # dotenv environment variables file
-.env
-.env.test
-.env.local
-.env_local
-.env_elephantsql
-.env_docker
```

Comment:
Might be a good idea to keep some of these in the gitignore, such as "env.test" or "env.local" to make local development easier